features/words/wordsSlice.js

```diff
@@ -0,0 +1,42 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { searchTypeEnum } from '../../app/decode/words'
+
+const initialState = {
+  chars: '',
+  searchType: searchTypeEnum.Subtring,
+  lenInterval: [3, 14],
```

Comment:
Jako uživatel se přimlouvám ze defaultní filtr, který zahrnuje všechna slova. Což by v kontextu toho, že lze aktuálně vyhledávat slova délky 1 až 14, reprezentoval interval [1, 14]. Pokud bylo cílem 3 odfiltrovat slova s délkou menší než 3, tak si myslím, že se to stane většinou přiorozeně tím, že uživatel zadá postupně víc než 3 písmena.

Reply:
Myšlenka byla, že slova délky 1 a 2 typicky nevyhledáváš, takže v základu jen překáží a když si je chceš přidat, tak si posuneš. Každopádně jsem to změnil na 1, protože taková slova jsou stejně nejníž.